app/templates/components/job-tabs.hbs

```diff
@@ -2,10 +2,10 @@
   <ul>
     <li>
       <LinkTo
-        @route="job.index"
-        @models={{array this.repo this.job}}
+        @route="{{this.route}}.index"
+        @models={{array this.repo this.model}}
```

Comment:
I didn't find any `route` properties on `job-tabs` component. Is it missing? If it's passed from external code, then there should be some kind of validation. In case of incorrect route name or models passed to `LinkTo` helper, it'll fail to generate the link and throw an exception, which'll break the entire app. Thus, I'd highly recommend to make this assignment as obvious and predictable as possible. E.g. I'd suggest to move definition of both route name and models to component file and map them explicitly to the input values (like `route1 -> routeName1 + models1, route2 -> routeName2 + models2`.